lib/sitemap.js

```diff
@@ -95,7 +95,7 @@ function SitemapItem(conf) {
   // The priority of this URL relative to other URLs
   // du to this field is optional no default value is set
   // please see: http://www.sitemaps.org/protocol.html
-  this.priority = conf['priority'];
+  this.priority = typeof conf['priority'] === 'number' ? conf['priority'] : 0.5;
```

Comment:
this probably has to be done like this to prevent evaluating `priority: 0` as `false` and therefore overwriting it with default value